packages/grafana-ui/src/components/uPlot/Plot.scss

```diff
@@ -9,11 +9,14 @@
   background: rgba(120, 120, 130, 0.2);
 }
 
-.u-cursor-x {
+.u-hz .u-cursor-x,
+.u-vt .u-cursor-y {
+  height: 100%;
```

Comment:
i think you can just omit `height` here, and `width` in the one below. we're just interested in the border color override, iiuc.